src/tsserver/server.ts

```diff
@@ -79,6 +79,16 @@ namespace ts.server {
     if (typeof process !== "undefined") {
         start(initializeNodeSystem(), require("os").platform());
     }
+    // TODO: Not sure this is right
+    else if (findArgument("vfs")) {

```

Comment:
`hasArgument`?

Reply:
Also, I don't know if we validate command line arguments but, if we do, we should probably warn/error if you pass this to a node-based server.